.github/workflows/coding.yml

```
@@ -0,0 +1,55 @@
+        run: npm ci
+        env:
+          TEST: true
+      - name: Check code
+        run: ls -all
+  
+  # PROGRAMMING LANGUAGE
+  # SCALA
+  build-scala-latest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)